src/main/scala/se/nimsa/dcm4che/streams/DicomFlows.scala

```diff
@@ -96,64 +96,64 @@ object DicomFlows {
 
   /**
     * Filter a stream of dicom parts such that all attributes except those with tags in the white list are discarded.
-    * Applies to headers and subsequent value chunks, DicomAttribute:s (as produced by the associated flow) and
-    * fragments. Sequences, items, preamble, deflated chunks and unknown parts are not affected.
     *
-    * @param tagsWhitelist list of tags to keep
-    * @param applyToFmi    if false, this filter does not affect the FMI
+    * @param tagsWhitelist list of tags to keep.
     * @return the associated filter Flow
     */
-  def partFilter(tagsWhitelist: Seq[Int], applyToFmi: Boolean = false) = whitelistFilter(tagsWhitelist.contains(_), applyToFmi)
+  def partFilter(tagsWhitelist: Seq[Int]) = whitelistFilter(tagsWhitelist.contains(_))
```

Comment:
Should we keep this alias for `whitelistFilter`? Perhaps the API will be cleaner with just one name?